### DISCUSSION TITLE
Invalid configuration option: includeForks

### DISCUSSION THREAD

**Discussion by Zoey2936** [renovatebot/renovate#20964] [Request Help]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

> There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.

> Location: `renovate.json`
> Error type: The renovate configuration file contains some invalid settings
> Message: `Invalid configuration option: includeForks`

Renovate created issues with this message in all repos, with includeForks enabled inside the renovate.json. It worked until now without problems. I couldn't find something like this inside the changelogs.

**Comment by rarkins** [comment]
Thanks for reporting. Fix pending in https://github.com/renovatebot/renovate/pull/20965
